tests/kernel/profiling/profiling_api/prj.conf

```diff
@@ -8,8 +8,9 @@ CONFIG_TEST_HW_STACK_PROTECTION=n
 # to check idle thread
 CONFIG_SYS_POWER_MANAGEMENT=y
 CONFIG_SYS_POWER_LOW_POWER_STATES=y
-CONFIG_IDLE_STACK_SIZE=512
+CONFIG_SYS_PM_POLICY_APP=y
+CONFIG_IDLE_STACK_SIZE=768
 
 # to check isr
 CONFIG_IRQ_OFFLOAD=y
-CONFIG_ISR_STACK_SIZE=512
+CONFIG_ISR_STACK_SIZE=768
```

Comment:
was increasing of isr stack needed?

Reply:
I followed the changes in the idle there as it looks that we were on the edge there.